nilearn/decoding/space_net_solvers.py

```diff
@@ -207,11 +230,12 @@ def _logistic_derivative_lipschitz_constant(X, mask, grad_weight,
     grad_buffer = np.zeros(mask.shape)
     for _ in range(n_iterations):
         grad_buffer[mask] = a
-        a = - _div(_gradient(grad_buffer))[mask] / sqrt(np.dot(a, a))
+        a = -_div(_gradient(grad_buffer))[mask] / sqrt(np.dot(a, a))
 
     grad_buffer[mask] = a
-    grad_constant = (- np.dot(_div(_gradient(grad_buffer))[mask], a)
-                     / np.dot(a, a))
+    grad_constant = -np.dot(_div(_gradient(grad_buffer))[mask], a) / np.dot(
+        a, a
```

Comment:
small rant: this is not super readable

Reply:
yeah that's the problem with black as a formatter, it is VERY opinionated.

I could add a `fmt: off` to those sections to tell black to leave it alone.

Reply:
It's just my opinion, but I wouldn't use `fmt: off` anywhere.
I think the whole purpose of using an automated formater is to never deal with formatting again. Indeed, there will be some ugly newlines, but overall the codebase will be much more readable, so I wouldn't care too much about these.

Reply:
yeah that's be my opinion too, but I wanted to make sure people are of other possibilities so we can make the best informed collective decision. 😉

Reply:
What we could do though is to increase the line length to the default line length in black and disable flake8's error. I guess this would significantly reduce the number of ugly lines.

Reply:
I don't have a strong opinion here. We can bring it up at the next developers meeting in case someone has a strong argument to keep it as is. I guess this will also be a good opportunity to discuss other formatting choices or changes to move forward with before reformatting the whole codebase